### ISSUE TITLE
metadata dataset with occurrences?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4091]
**metadata dataset with occurrences?**

I suspect this dataset type is incorrect, and someone could perhaps inform the publisher?

-----

Github user: @dnoesgaard
User: [See in registry](https://www.gbif.org/api/feedback/user/7df5daea1528c57a2f7cb52d3b459013:afb4f74a5addbdf4c32dcfc389504d2393856cb8452d1728ee3bcdaf82b7549adb3066f1292322d852a66e5ae68f21ac2daf827092627cf79c46503e214034295a200b8d23d422fe3a85f67ac9641001320768b2d862692e7b64a1367950faa33c09c11f731f2fb0746509c53a79e89de27896e0fed3b7928eccd2f437313f203a6e4c294cae7b5b1dbfac0188b97ceb) - [Send email](https://www.gbif.org/api/feedback/user/mailto/7df5daea1528c57a2f7cb52d3b459013:afb4f74a5addbdf4c32dcfc389504d2393856cb8452d1728ee3bcdaf82b7549adb3066f1292322d852a66e5ae68f21ac2daf827092627cf79c46503e214034295a200b8d23d422fe3a85f67ac9641001320768b2d862692e7b64a1367950faa33c09c11f731f2fb0746509c53a79e89de27896e0fed3b7928eccd2f437313f203a6e4c294cae7b5b1dbfac0188b97ceb)
System: Chrome 101.0.4951 / Mac OS X 10.15.7
Referer: https://www.gbif.org/dataset/dc878d3c-ebc2-4b95-824b-2209fb3f38f4
Window size: width 1512 - height 748
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-05-20T08:05:07.675Z',to:'2022-05-20T08:11:07.675Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-05-20T08:05:07.675Z',to:'2022-05-20T08:11:07.675Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
@dnoesgaard it says "metadata-only" because all the occurrences were removed from the dataset. But because of our safeguard, they haven't been deleted from our index. I wanted to check if there was any comment on the IPT side explaining why the occurrences have been removed but they seem to have some issue: https://ipt.pensoft.net/resource?r=database_fish_urban_tsurumi. I have contacted Pensoft to ask if they can take a look.
